Eatery/Controllers/Eateries/CollegetownEateriesMenuViewController.swift

```diff
@@ -0,0 +1,389 @@
+//
+//  CollegetownMenuViewController.swift
+//  Eatery
+//
+//  Created by Gonzalo Gonzalez on 3/3/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Crashlytics
+import MapKit
+import UIKit
+
+class CollegetownEateriesMenuViewController: UIViewController, UIScrollViewDelegate, MKMapViewDelegate, CLLocationManagerDelegate {
+    
+    var eatery: CollegetownEatery
+    var delegate: MenuButtonsDelegate?
+    var userLocation: CLLocation?
+    
+    var outerScrollView: UIScrollView!
+    var backButton: UIButton!
+    var ctownMenuHeaderView: CollegetownMenuHeaderView!
+    var informativeViews: [UIView]!
```

Comment:
If this array always has three elements, split this into three properties and work with those. If there is common set up functionality required between all three, then use a function for that.